src/components/tabset/tabset.component.html

```diff
@@ -15,7 +15,28 @@
         [class.disabled]="tab.disabled"
         [ngClass]="tab.customClass">
 
-        <a class="nav-link"
+       <ng-container *ngIf="tab.routerPath">
+            <a class="nav-link"
+            [id]="tab.id"
+            role="tab"
+            uxTabbableListItem
+            uxFocusIndicator
+            (mousedown)="_tabset.select(tab)"
+            (activated)="_tabset.select(tab)"
+            [attr.aria-controls]="tab.id"
+            [attr.aria-selected]="(_tabset.activeTab$ | async) === tab"
+            [attr.aria-disabled]="tab.disabled"
+            [routerLink]="tab.routerPath">
+
+            <span *ngIf="!tab.headingRef">{{ tab.heading }}</span>
+
+            <ng-container *ngIf="tab.headingRef" [ngTemplateOutlet]="tab.headingRef"></ng-container>
```

Comment:
This should probably be refactored as well, even though it's only two lines at the moment.